ush/machine/wcoss2.yaml

```diff
@@ -9,8 +9,8 @@ platform:
   MET_BIN_EXEC: bin
   MET_INSTALL_DIR: /apps/ops/para/libs/intel/19.1.3.304/met/10.1.1
   DOMAIN_PREGEN_BASEDIR: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/FV3LAM_pregen
-  QUEUE_DEFAULT: dev
-  QUEUE_FCST: dev
+  QUEUE_DEFAULT: devhigh
+  QUEUE_FCST: devhigh
```

Comment:
@JianpingHuang-NOAA, please do not change `QUEUE_DEFAULT` and `QUEUE_FCST` to "devhigh" here. Since only a few users can use "devhigh", it is not a good idea to put "devhigh" by default. You can change them in the configuration file separately. We can put them by default later when we deliver aqm.v7 to NCO.

Reply:
Changed back